.github/workflows/hardhat-ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: 18
+          node-version: 20
```

Comment:
The prebuild scripts that generate the contract types, and allow us to then do a full build use Hardhat.
As Hardhat is ESM, running it under Node 18 fails.
As `./packages` are going away I preferred to remove Node 18 from the test matrix.